docs/reference/commandline/build.md

```diff
@@ -251,3 +252,22 @@ flag](/reference/run/#specifying-custom-cgroups).
 Using the `--ulimit` option with `docker build` will cause each build step's 
 container to be started using those [`--ulimit`
 flag values](/reference/run/#setting-ulimits-in-a-container).
+
+Currently, you can use `ENV` instructions in a Dockerfile to define variable
```

Comment:
@thaJeztah I'm fine with the comment. Adding the heading would call it out but we don't categorize all the other examples.  If we just arbitrarily add a heading ... Let me create a beginner issue to sort that out, and see what we get.